### ISSUE TITLE
React Native 0.74.0 - Getting issue on production build -Crashed: Thread: SIGSEGV  0x0000006df6aebf90

### ISSUE THREAD

**Issue by pspsingh677** [facebook/react-native#46158]
### Description

I am trying to reproduce in debug and release apk file but not getting this crash only getting play store app, App is randomly crashed. This crash is occur in most devices one-plus and samsung devices

### Steps to reproduce

I am trying to reproduce in debug and release apk file but not getting this crash only getting play store app, App is randomly crashed.

### React Native Version

0.74.0

### Affected Platforms

Runtime - Android

### Output of `npx react-native info`

```text
System:
  OS: macOS 14.6.1
  CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Memory: 21.85 MB / 16.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 18.20.3
    path: /usr/local/bin/node
  Yarn: Not Found
  npm:
    version: 10.8.1
    path: ~/Documents/twc-reactnative/node_modules/.bin/npm
  Watchman:
    version: 2024.07.08.00
    path: /usr/local/bin/watchman
Managers:
  CocoaPods: Not Found
SDKs:
  iOS SDK: Not Found
  Android SDK: Not Found
IDEs:
  Android Studio: 2024.1 AI-241.15989.150.2411.11948838
  Xcode:
    version: /undefined
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 17.0.10
    path: /usr/bin/javac
  Ruby:
    version: 2.6.10
    path: /usr/bin/ruby
npmPackages:
  "@react-native-community/cli":
    installed: 13.6.9
    wanted: 13.6.9
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.74.0
    wanted: 0.74.0
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: true
  newArchEnabled: false
```

### Stacktrace or Logs

```text
Crashed: Thread: SIGSEGV  0x0000006df6aebf90
#00 pc 0x95f88 libc.so (BuildId: d1a98b526f2f94260a53c3055979a4f6)
#01 pc 0x2e064 libopenjdk.so (BuildId: c5d0251899adde41ff751e77fb677ec7)
#02 pc 0x70e97448
#03 pc 0x5b7704 libart.so (BuildId: 2452917c4ff69cbb6e75e5512260946b)
#04 pc 0x5af57c libart.so (BuildId: 2452917c4ff69cbb6e75e5512260946b)
#05 pc 0x70e976dc
```

### Reproducer

https://github.com/react-native-community/reproducer-react-native

**Comment by pspsingh677** [comment]
https://github.com/facebook/react-native/issues/46158#issuecomment-2304371346

> ⚠️	Newer Version of React Native is Available!
> ℹ️	You are on a supported minor version, but it looks like there's a newer patch available - 0.74.5. Please [upgrade](https://reactnative.dev/docs/upgrading) to the highest patch for your minor or latest and verify if the issue persists (alternatively, create a new project and repro the issue in it). If it does not repro, please let us know so we can close out this issue. This helps us ensure we are looking at issues that still exist in the most recent releases.



> ⚠️	Newer Version of React Native is Available!
> ℹ️	You are on a supported minor version, but it looks like there's a newer patch available - 0.74.5. Please [upgrade](https://reactnative.dev/docs/upgrading) to the highest patch for your minor or latest and verify if the issue persists (alternatively, create a new project and repro the issue in it). If it does not repro, please let us know so we can close out this issue. This helps us ensure we are looking at issues that still exist in the most recent releases.

I have checked with 0.74.5 but same issue facing

**Comment by cortinico** [comment]
> Crashed: Thread: SIGSEGV  0x0000006df6aebf90
> #00 pc 0x95f88 libc.so (BuildId: d1a98b526f2f94260a53c3055979a4f6)
> #01 pc 0x2e064 libopenjdk.so (BuildId: c5d0251899adde41ff751e77fb677ec7)
> #02 pc 0x70e97448
> #03 pc 0x5b7704 libart.so (BuildId: 2452917c4ff69cbb6e75e5512260946b)
> #04 pc 0x5af57c libart.so (BuildId: 2452917c4ff69cbb6e75e5512260946b)
> #05 pc 0x70e976dc

This stacktrace is unrelated to React Native.

Sadly without a reproducer or any more information, there is nothing we cannot do here.
